src/epub/text/chapter-27.xhtml

```diff
@@ -14,7 +14,7 @@
 of some misunderstanding. Your kind offices will set all right: he is the only man I ever did or could love, and I trust you will convince him of it. The spring fashions are partly down; and the hats the most frightful you can imagine. I hope you spend your time pleasantly, but am afraid you never think of me. I will not say all that I could of the family you are with, because I would not be ungenerous, or set you against those you esteem; but it is very difficult to know whom to trust, and young men never know their minds two days together. I rejoice to say that the young man whom, of all others, I particularly abhor, has left Bath. You will know, from this description, I must mean Captain Tilney, who, as you may remember, was amazingly disposed to follow and tease me, before you went away. Afterwards he got worse, and became quite my shadow. Many girls might have been taken in, for never were such attentions; but I knew the fickle sex too well. He went away to his regiment two days ago, and I trust I shall never be plagued with him again. He is the greatest coxcomb I ever saw, and amazingly disagreeable. The last two days he was always by the side of Charlotte Davis: I pitied his taste, but took no notice of him. The last time we met was in Bath Street, and I turned directly into a shop that he might not speak to me; I would not even look at him. He went into the pump room afterwards; but I would not have followed him for all the world. Such a contrast between him and your brother! Pray send me some news of the latter⁠—I am quite unhappy about him; he seemed so uncomfortable when he went away, with a cold, or something that affected his spirits. I would write to him myself, but have mislaid his direction; and, as I hinted above, am afraid he took something in my conduct amiss. Pray explain everything to his satisfaction; or, if he still harbours any doubt, a line from himself to me, or a call at Putney when next in town, might set all to rights. I have not been to the rooms this age, nor to the play, except going in last night with the Hodges, for a frolic, at half price: they teased me into it; and I was determined they should not say I shut myself up because Tilney was gone. We happened to sit by the Mitchells, and they pretended to be quite surprised to see me out. I knew their spite: at one time they could not be civil to me, but now they are all friendship; but I am not such a fool as to be taken in by them. You know I have a pretty good spirit of my own. Anne Mitchell had tried to put on a turban like mine, as I wore it the week before at the concert, but made wretched work of it⁠—it happened to become my odd face, I believe, at least Tilney told me so at the time, and said every eye was upon me; but he is the last man whose word I would take. I wear nothing but purple now: I know I look hideous in it, but no matter⁠—it is your dear brother’s favourite colour. Lose no time, my dearest, sweetest Catherine, in writing to him and to me,</p>
```

Comment:
The Pump Room is a historic named place so we should modernize `Pump-room` to `Pump Room`